DuckDuckGo/ConfigurationURLDebugViewController.swift

```diff
@@ -127,7 +127,7 @@ final class ConfigurationURLDebugViewController: UITableViewController {
         cell.subtitle.text = url(for: row)
         cell.subtitle.textColor = customURL(for: row) != nil ? UIColor(designSystemColor: .accent) : .black
         cell.ternary.text = lastConfigurationUpdateDate != nil ? dateFormatter.string(from: lastConfigurationUpdateDate!) : "-"
-        cell.refresh.addAction(makeAction(for: row), for: .allEvents)
+        cell.refresh.addAction(refreshAction, for: .primaryActionTriggered)
```

Comment:
The previous implementation was causing an ever increasing number of actions to be added to the button.

This change fixes two things:
- We no longer create an infinite number of new actions and add them.
- We no longer trigger the action 3 times (`.allEvents`).